docs/intro.mdx

```diff
@@ -67,30 +67,38 @@ around identities, sessions, login, OAuth2, permissions, and more. The core comp
 
 Ory Network incorporates the open-source [Ory Kratos Identity Server](https://www.ory.sh/kratos) and offers:
 
-- **Self-service** flows are everything users do on their own / without the help of others:
-- Registration with passwords, social sign-in, OpenID Connect, ...
-- Login with passwords, social sign-in, OpenID Connect, ...
-- Updating the profile, email, changing the password, un/linking with social sign-in providers, ...
+- Self-service flows are everything users do on their own / without the help of others:
+- Registration with passwords, social sign-in, OpenID Connect, passkeys, and more.
+- Login with passwords, social sign-in, OpenID Connect, passkeys, and more.
+- Updating the profile, email, changing the password, un/linking with social sign-in providers, and more.
 - Recovering the account by resetting the password.
-- Verifying email addresses, phone numbers,...
+- Verifying email addresses, phone numbers, and more.
 - Multi-factor authentication flows and recovery processes.
-- **Administrative identity management** to get, create, update, and delete identities (users) and their data.
-- **Headless APIs and data models** allow you to fully customize Identity Schemas (for example adding fields like first name,
-  accept tos, shipping address, gender, ...) and create your login, registration, profile settings, recovery, and verification
-  screen using SDKs and REST APIs.
+- Administrative identity management to get, create, update, and delete identities and their data.
+- Headless APIs and data models allow you to fully customize Identity Schemas - for example adding fields like name, accept ToS,
+  phone number - and create your login, registration, profile settings, recovery, and verification screen using SDKs and REST
+  APIs.
 
 ### Permissions and relationships
 
 Ory Network incorporates the open-source [Ory Keto Permission Server](https://www.ory.sh/keto) and offers:
 
-- **Permission management** to get, create, update, and delete permissions.
-- **Permission checking** to check if a user has a permission.
+- Permission management to get, create, update, and delete permissions.
+- Permission checking to check if a user has a permission.
 
 ### OAuth2 and OIDC
 
 Ory Network incorporates the open-source [Ory Hydra OAuth2 & OpenID Server](https://www.ory.sh/hydra) and offers:
 
-- **Fully featured OAuth2 & [OpenID Certified](https://openid.net/developers/certified/)® OIDC Provider**
+- Fully featured OAuth2 & [OpenID Certified](https://openid.net/developers/certified/)® OIDC Provider
+
+### SAML and SCIM
+
+Ory Network incorporates the open-source [Ory Polis](https://www.ory.sh/polis) and offers:
+
+- Enterprise SSO integration with SAML and OIDC identity providers such as Okta, Azure AD, and Google Workspace.
```

Comment:
nit: OIDC integration was already in kratos. Polis brings SAML SP and SAML IDP (soon to be released)